Simple.OData.Client.V3.Adapter/ResponseReader.cs

```diff
@@ -30,10 +30,14 @@ public override Task<ODataResponse> GetResponseAsync(HttpResponseMessage respons
 #if SILVERLIGHT
         public async Task<ODataResponse> GetResponseAsync(IODataResponseMessage responseMessage)
 #else
+
         public async Task<ODataResponse> GetResponseAsync(IODataResponseMessageAsync responseMessage)
 #endif
         {
-            var readerSettings = new ODataMessageReaderSettings();
+            var readerSettings = new ODataMessageReaderSettings
+            {
+                UndeclaredPropertyBehaviorKinds = ODataUndeclaredPropertyBehaviorKinds.IgnoreUndeclaredValueProperty
```

Comment:
Should be simple to hoist this into either (a) a new property on `ODataClientSettings`, or (b) use `ISession.Settings.IgnoreUnmappedProperties` to choose this value. This would enable those of us who need strict handling to use a version of the client with this patch.